pkg/hintrunner/hint.go

```diff
@@ -1278,3 +1277,58 @@ func (hint *AssertLeIsSecondArcExcluded) Execute(vm *VM.VirtualMachine, ctx *Hin
 
 	return vm.Memory.WriteToAddress(&addr, &writeValue)
 }
+
+type RandomEcPoint struct {
+	x CellRefer
+	y CellRefer
+}
+
+func (hint *RandomEcPoint) String() string {
+	return "RandomEc"
+}
+
+func (hint *RandomEcPoint) Execute(vm *VM.VirtualMachine) error {
+	// Keep sampling a random field element `X` until `X^3 + X + beta` is a quadratic residue.
+	betaFelt := f.Element{3863487492851900874, 7432612994240712710, 12360725113329547591, 88155977965380735}
```

Comment:
Where do these values come from?
If they are from an external source it would be nice to add it here as a comment